src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@use './variables' as *;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__main-image {
+    margin: 32px auto 40px;
+  }
+
+  &__item--name {
+    margin-bottom: 4px;
+  }
+
+  &__item--description {
+    margin-bottom: $margin-main;
+  }
+
+  &__wrapper {
+    margin-bottom: $margin-main;
+  }
+
+  &__button {
+    margin-bottom: $margin-main;
+  }
+}
+
+.main-image {
+  background-image: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
The '.wrapper' class seems to be a standalone block, but it's not clear if it is related to '.product' or not. If it's an element of '.product', it should follow BEM naming convention as '.product__wrapper'.